pipelines/LSSTCam/cpFlatSingleChip.yaml

```diff
@@ -1,4 +1,4 @@
-description: cp_pipe FLAT calibration construction
+description: cp_pipe LSSTCam FLAT calibration construction, optimized for a single chip.
```

Comment:
This file can probably be dropped, as we're unlikely to need it for the full camera.

Reply:
I assumed we wanted these in case somebody wanted to run a test flat for a single chip.  If that's wrong I can remove them all.